tests/test_config.py

```diff
@@ -345,5 +345,23 @@ def test_get_hdfs_to_snowflake_query_creates_query(self):
             """.strip())
 
 
+class TableSchemaTests(ConfigTests):
+    def setUp(self):
+        self.result = f"""
+SELECT column_name AS "name", data_type AS "originalType"
+FROM information_schema.columns
+WHERE table_name = 'B'
+  AND table_schema = 'A'
+        """.strip()
+
+    def test_get_table_schema_sql_1(self):
```

Comment:
The names of test methods should say what it is testing. You can see this in any other test method in this file. "`_1`" doesn't mean anything.